src/functions/template-tags/multiday.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Calendar Multiday Functions
+ *
+ * Display functions (template-tags) for use in WordPress templates.
+ */
+
+// Don't load directly
+if ( ! defined( 'ABSPATH' ) ) {
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 935d65cad5615561c74dab57d06ef067</sub>